logos/benchmarks/benchmark_logos.py

```
@@ -706,8 +717,75 @@ async def _run(entry: WorkloadEntry, start_mono: float) -> None:
+                    r = task.result()
+                    if r.success:
+                        ttft = f"TTFT={r.ttft_ms:.0f}ms" if r.ttft_ms else "no TTFT"
+                        print(f"  [warmup] {model:<{width}}  OK      {ttft}")
+                    else:
+                        print(f"  [warmup] {model:<{width}}  FAIL    {r.status_code} {(r.error or '')[:60]}")
+                except Exception as exc:
+                    print(f"  [warmup] {model:<{width}}  ERROR   {exc}")
+            else:
+                print(f"  [warmup] {model:<{width}}  TIMEOUT")
+
+    print(f"  [warmup] Done.")
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)